### PR TITLE
chore: update readme about development one-time passwords

### PR DESCRIPTION
**Pull request by jimvae**
## Problem

The information about one-time passwords (OTPs) in readme for development is outdated. Maildev is currently being used to obtain OTPs

## Solution

Updated readme to include information about obtaining OTPs through maildev for development

### CODE REVIEW

**Review by yong-jie — APPROVED** [review]
lgtm!

File: README.md [hunk 1]
```diff
@@ -71,10 +71,8 @@ Note that 3000 is the port number that the webpack dev server listens on; the ba
 on port 8080 instead.
 
 Because redirects are served directly from the backend, shortlinks can be accessed via `localhost:3000/shortlink`, 
-but that is really being proxied to `localhost:8080/shortlink`. Also, given that GoGovSG will attempt to send
-emails directly from your computer when running on localhost, there is a chance that the email might land in
-spam or not be sent entirely. To mitigate this, we have set the one-time password for all log-in attempts
-on localhost to be `111111`.
+but that is really being proxied to `localhost:8080/shortlink`. One-time passwords for all log-in attempts on localhost
+are obtained using [maildev](https://github.com/maildev/maildev) and accesed via `http://localhost:1080/`.
```
Comment (halfwhole) [inline README.md:75]: oh no should be `accessed`

Comment (yong-jie) [reply README.md:75]: ahhhhh crap, lets get another PR open to fix that typo

Comment (jimvae) [reply README.md:75]: Sorry for this, fixed it in in this [PR](https://github.com/opengovsg/GoGovSG/pull/1923)

**Pull request merged**